components/src/dynamo/profiler/tests/unit/test_helpers_profile_sla.py

```
@@ -546,178 +565,597 @@ def test_mocker_plus_planner_uses_mocker_base(self, tmp_path):
                 PickedParallelConfig(tp=1),
             )
 
-        mock_mocker.assert_called_once()
-        mock_planner.assert_called_once()
-        assert mock_planner.call_args.args[1] is mocker_base
-        assert result == [planner_cm, profile_cm, mocker_base]
+        assert result is mocker_cfg
+
+
+# ---------------------------------------------------------------------------
+# optimizationType – SLASpec validation
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)